docs/source/quickstart.py

```diff
@@ -0,0 +1,14 @@
+'''
+# init
+>>> from opendp.mod import enable_features
+>>> enable_features('contrib')
```

Comment:
I'm surprised to see this all inside a string! Do you think it's worth keeping the code inside strings to get the `>>>` formatted doctests?

Reply:
I do like the doctest style, but could reconsider.

for doctest:
- Particularly if moving away from notebooks, it would be a loss to not show any output.
- Could add `assert` but I find that a little clunkier, particularly if the output is longer. 
- If it was just python code, we'd need machinery to run it, I'm not sure it would work as smoothly with pytest.
- Sphinx renders this nicely, and you can copy and paste without getting the `>>>` prefix.

for plain python:
- syntax highlighting during development
- runnable without doctest

Reply:
Maybe there's a way to get syntax highlighting in doctests?
Or maybe the file doesn't need to be `.py` if it's just a string?